components/org.wso2.carbon.identity.oidc.session/src/main/java/org/wso2/carbon/identity/oidc/session/util/OIDCSessionManagementUtil.java

```diff
@@ -246,7 +247,12 @@ public static Cookie removeOPBrowserStateCookie(HttpServletRequest request, Http
                                     MultitenantConstants.SUPER_TENANT_DOMAIN_NAME.equals(tenantDomain)) {
                                 servletCookie.setPath("/");
                             } else {
-                                servletCookie.setPath(FrameworkConstants.TENANT_CONTEXT_PREFIX + tenantDomain + "/");
+                                if (PrivilegedCarbonContext.getThreadLocalCarbonContext().getOrganizationId() != null) {
+                                    servletCookie.setPath("/o/" + tenantDomain + "/");
+                                } else {
+                                    servletCookie.setPath(
+                                            FrameworkConstants.TENANT_CONTEXT_PREFIX + tenantDomain + "/");
+                                }
```

Comment:
This fix is added as the opbs cookie removing path is set to tenanted path. Hence the cookie doesn't get cleared. Hence the following warn log should have been logged in the server log. With this fix, it was not reproduced.

<img width="1430" alt="Screenshot 2023-11-14 at 20 20 25" src="https://github.com/wso2-extensions/identity-inbound-auth-oauth/assets/35717390/f0edd0d7-edf4-4abb-b1fe-e5bcf172978f">